tests/introspection/ApiSelectorTest.cs

```diff
@@ -193,6 +193,13 @@ protected virtual bool Skip (Type type, string selectorName)
 				if (Runtime.Arch == Arch.SIMULATOR)
 					return true;
 				break;
+			case "ARBodyTrackingConfiguration":
+			case "ARGeoTrackingConfiguration":
```

Comment:
works on `ARWorldTrackingConfiguration` ?

Reply:
The test doesn't complain there, yeah that was a bit weird but device is fine so 🤷‍♂️